tls/s2n_async_pkey.c

```diff
@@ -409,3 +407,102 @@ S2N_RESULT s2n_async_pkey_sign_free(struct s2n_async_pkey_op *op)
 
     return S2N_RESULT_OK;
 }
+
+int s2n_async_get_op_type(struct s2n_async_pkey_op *op, s2n_async_pkey_op_type * type)
+{
+    POSIX_ENSURE_REF(op);
```

Comment:
You've got the pkcs11 tests as basically functional/integration tests, but they cover all your code? Do you need unit tests?